applications/asset_tracker_v2/src/modules/Kconfig.cloud_module

```diff
@@ -12,8 +12,8 @@ if CLOUD_MODULE
 
 config CLOUD_THREAD_STACK_SIZE
 	int "Cloud module thread stack size"
-	default 4096 if NRF_CLOUD
-	default 2688
+	default 6144 if NRF_CLOUD_MQTT
```

Comment:
Just out of curiousity: What did we change NRF_CLOUD to NRF_CLOUD_MQTT in this PR? And why did the default stack size increase? If this is only because of debug module, then shouldn't we increase the thread size only if the debug module is configured?

Reply:
After https://github.com/nrfconnect/sdk-nrf/pull/4609. Kconfigs associated with `NRF_CLOUD`  were split into separate module Kconfigs. `CLOUD_THREAD_STACK_SIZE` is now ultimately dependent on `NRF_CLOUD_MQTT` and not `NRF_CLOUD`. `NRF_CLOUD` is now more of a umbrella config for everything related to nRF Cloud libraries. We only care about the MQTT part in this case.

You`re right! The stack increase should depend on `CONFIG_DEBUG_MODULE_USE_CUSTOM_TRANSPORT` being enabled.